templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_member_list.html.twig

```diff
@@ -112,10 +112,10 @@
                     </th>
                     <th class="u-pt-0_5 u-pl-0_5 u-pb-0_25 text-right">
                         {{ 'invitation'|trans }}
-                        <dp-tooltip-icon
-                            icon="fa-question-circle"
-                            text="{{ 'invitable_institutions.invitation.explanation'|trans }}"
-                        ></dp-tooltip-icon>
+                        <dp-contextual-help
+                            icon="question"
```

Comment:
The `icon` prop of DpContextualHelp defaults to `question`, so this can be removed here.

Reply:
Done here [be33bc2](https://github.com/demos-europe/demosplan-core/pull/2376/commits/be33bc2d4b3ad5fce405dfbb868049c07e0673ee)